### PR TITLE
CMI MTE support - Docs changes

### PR DESCRIPTION
**Pull request by ravikiran-sulikeri**
[Trello](https://trello.com/c/LjlhH3N5/2273-nom-metrics-support-for-business-critical-mte-in-cmi)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-463.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-463.surge.sh/aura/platform/metrics-integration)

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 1]
```diff
@@ -4,9 +4,16 @@
 
 label:AuraDB-Virtual-Dedicated-Cloud[]
 label:AuraDS-Enterprise[]
+label:AuraDB-Business-Critical[]
+
+An application performance monitoring system can be configured to fetch metrics of AuraDB instances of types:
+
+* Virtual Dedicated Cloud
```
Comment (alexnb) [inline modules/ROOT/pages/platform/metrics-integration.adoc:11]:
```suggestion
* AuraDB Virtual Dedicated Cloud
```

**Review by alexnb — APPROVED** [review]
:+1: 

just a nit regarding consistent naming

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-463.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-463.surge.sh/aura/platform/metrics-integration)

**Review by AlexicaWright — APPROVED** [review]
LGTM!

**Pull request merged**